### PR TITLE
CHANGELOG.md: Add 0.4.0 entries

### PR DESCRIPTION
**Pull request by dmajda**

### CODE REVIEW

**Comment by dmajda** [comment]
@bkutil @demmer

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,20 +2,35 @@
...
 [[#242](https://github.com/juttle/juttle/pull/242)]
-- Create `http_server` adapter. [[#224](https://github.com/juttle/juttle/pull/224)]
+- Added the `-format` option to `read http` and `write file`. [[#287](https://github.com/juttle/juttle/issues/287), [#125](https://github.com/juttle/juttle/issues/125)]
+- Optimized `read ... | head ...` with `file`, `stdio`, and `http` adapters. [[#261](https://github.com/juttle/juttle/issues/261), [#267](https://github.com/juttle/juttle/pull/267), [#278](https://github.com/juttle/juttle/pull/278), [#279](https://github.com/juttle/juttle/pull/279)]
+- Made `emit -points` not add the `time` field to points that don’t contain it already. [[#77](https://github.com/juttle/juttle/issues/77)]
+- Made the `-points` option of `emit` mutually exclusive with `-from`, `-to`, `-last`, `-limit`, and `-every`. [[#274](https://github.com/juttle/juttle/pull/274)]
+- Made the `node-grok` dependency optional. [[#290](https://github.com/juttle/juttle/pull/290)]
+- Replaced JSHint + JSCS by ESLint and fixed various issues ESLint found. [[#202](https://github.com/juttle/juttle/issues/202)]
```
Comment (demmer) [inline CHANGELOG.md]: I think it'd be worth mentioning that we also added enforcement of single quotes (#238) and global strict mode (#249).

**Comment by demmer** [comment]
Other than that one comment, looks good to me

File: CHANGELOG.md (hunk 1 above)
Comment (dmajda) [inline CHANGELOG.md]: OK, will update (although single quote enforcement sounds mostly as an internal thing)

**Comment by dmajda** [comment]
OK, updated, merging.

**Pull request merged**